composer.json

```diff
@@ -12,12 +12,12 @@
     "type": "library",
     "autoload": {
         "psr-4": {
-            "OpenAPIValidation\\": "src/"
+            "League\\OpenAPIValidation\\": "src/"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "OpenAPIValidationTests\\": "tests/"
+            "League\\OpenAPIValidationTests\\": "tests/"
```

Comment:
@lezhnev74 how about `League\OpenAPIValidation\Tests\` for test to have consistent `League\OpenAPIValidation` as base namespace?

Reply:
@scaytrase Imho it should be fine either way. Afaics, all league libraries do it differently, for example:

https://github.com/thephpleague/flysystem/blob/master/tests/AdaptersThatCanOverwriteFilesTest.php
https://github.com/thephpleague/tactician/blob/master/tests/CommandBusTest.php
https://github.com/thephpleague/oauth2-server/blob/master/tests/AuthorizationServerTest.php

Reply:
Yeah, good point. Checked another package, it does the same naming:
https://github.com/thephpleague/commonmark/blob/master/composer.json

Reply:
@lezhnev74 / @scaytrase  would you prefer changing it that way?